stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/extensions/filter.py

```diff
@@ -171,6 +183,16 @@ def to_es(self):
                     "must_not": [{"term": {to_es(self.args[0]): to_es(self.args[1])}}]
                 }
             }
+        elif self.op == AdvancedComparisonOp.like:
+            return {
+                "wildcard": {
+                    to_es(self.args[0]): {
+                        "value": cql2_like_to_es(str(to_es(self.args[1]))),
+                        "boost": 1.0,
```

Comment:
why is boost used here?

Reply:
`"boost": 1.0` is the default value and not needed. Removed in the latest commit.

Reply:
iirc, boost only applies when doing the type of search where the matches are rated for relevance (I can't remember what that's called), whereas we're always doing filtered searches